metahtml/timestamp.py

```diff
@@ -337,21 +337,40 @@ def get_timestamp_published(html, url, **kwargs):
         ( 'bles.com',                       '(//span[@class="p-time"]/@data-date)[1]' ),
         ( 'blogs.wsj.com',                  '//meta[@name="article.published"]/@content' ),
         ( 'bloomberg.com',                  '//meta[@name="parsely-pub-date"]/@content' ),
+        ( 'breitbart.com',                  '//meta[@name="pubdate"]/@content' ),
         ( 'cbsnews.com',                    '//time/@datetime' ),
-        ( 'chicagotribune.com',             '//meta[@name="date"]/@content'),
```

Comment:
I can see that you've modified this line to put an additional space before the `)`.  In general, you shouldn't modify code in a project that is not directly related to what you're working on.  So in the future, you should consider existing lines as "read only" even if they contain formatting mistakes.

The most correct way to modify these sorts of lines is to use a separate commit that contains only those modifications so that I can see exactly which changes are stylistic and which are substantive.